### PR TITLE
#1221 Observations

### PR DESCRIPTION
**Pull request by kenzo23312**

### CODE REVIEW

File: frontend/src/components/partners/profile/icons/observationTypeIcon.js [hunk 1]
```diff
@@ -1,27 +1,88 @@
...
 import ObservationIcon from './observationIcon';
 import FlagIcon from './flagIcon';
 import EscalatedIcon from './escalatedIcon';
+import ProjectPartners from '../../../forms/fields/projectFields/partnersField/ProjectPartners';
+
+const styleSheet = theme => ({
+  flagBg: {
+    marginLeft: '5px',
+    paddingLeft: '2px',
+    paddingRight: '2px',
```
Comment (momimomo) [inline frontend/src/components/partners/profile/icons/observationTypeIcon.js]: padding: '0 2px',

File: frontend/src/components/partners/profile/modals/updateObservationEscalated/updateEscalatedObservationForm.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+export const ESCALATION_DECISION = {
+  DEFFERED: 'DB',
+  CONFIRMED: 'EF',
+};
+
+const radioFlag = [
+  {
+    value: ESCALATION_DECISION.DEFFERED,
+    label: <div style={{ display: 'flex', alignItems: 'center' }}><FlagIcon color={FLAGS.YELLOW} />
+      {'Deffered back to field or local decision making'}
```
Comment (momimomo) [inline frontend/src/components/partners/profile/modals/updateObservationEscalated/updateEscalatedObservationForm.js]: ???

File: frontend/src/components/partners/profile/icons/observationTypeIcon.js (hunk 1 above)
Comment (kenzo23312) [reply frontend/src/components/partners/profile/icons/observationTypeIcon.js]: done

File: frontend/src/components/partners/profile/modals/updateObservation/updateObservationForm.js [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+    label: 'This flag is no longer valid',
+  },
+  {
+    value: OBSERVATION_DECISION.ESCALATE,
+    label: 'Escalate to UN Headquarters Editor',
+  },
+];
+
+const commentFormControlStyle = {
+  paddingBottom: '12px',
```
[the remark this answers is not in the document]
Comment (kenzo23312) [reply frontend/src/components/partners/profile/modals/updateObservation/updateObservationForm.js]: done

**Pull request merged**